design/stackhub.md

```diff
@@ -0,0 +1,207 @@
+# Collection transition to Stack Design
+
+## Key Concepts / Background
+- Simplify key concepts by eliminating Collections and Collection Hubs
+  -  Object model inheritence, e.g. A Collection ISA a Stack and a CollectionHub ISA a Stackhub, causes confusion especially for community messsaging, developers and new comers to the project.
+  -  Monolithic build (CollectionHub) causes painful lifecycle for making configuration or manage the curated stacks and pipelines.
+
+- In general, to replace Collection Hubs with Stack Hubs, and providing configuration for the governed stacks via the defined Kubernetes Custom Resources:  Kabanero CR.
```

Comment:
I think stack hubs are probably an unnecessary concept. We have stacks, stacks have an index, the Kabanero CR should just point at one or more stack indexes as individual references. I think our message to Kabanero users should be that they build and manage stacks as individual entities (we can support a composite index for evaluation).

Reply:
Appsody Stackhubs is a term that is defined by Appsody.  I am happy to consider that an index.yaml represents 1:many stacks and that is all there is --- one has 1:many curated stacks.

Reply:
Steve, looks like Appsody refers to hub as "Appsody Hub".  https://appsody.dev/docs
 i googled Appsody "stackhub" - no hits.

Reply:
Appsody refers to the Appsody Hub, this is it's centrally managed location for contributed stacks. Appsody does not have the concept of stack hubs. There are two delivery models for a stack either it is contributed to the Appsody stacks repository and therefore included in the Appsody hub, or it is released as an individual stack with its own index.

The current form of the Appsody hub is simply a composite index and is a term that may be dropped form the Appsody vocabulary in future.